src/camera.ts

```diff
@@ -259,17 +293,29 @@ class Camera extends Element {
 
         // in with the new
         const pixelFormat = PIXELFORMAT_RGBA8;
+        const samples = this.scene.config.camera.multisample ? device.maxSamples : 1;
+
         const colorBuffer = createTexture(width, height, pixelFormat);
         const depthBuffer = createTexture(width, height, PIXELFORMAT_DEPTH);
         const renderTarget = new RenderTarget({
             colorBuffer: colorBuffer,
             depthBuffer: depthBuffer,
             flipY: false,
-            samples: this.scene.config.camera.multisample ? device.maxSamples : 1,
+            samples: samples,
             autoResolve: false
         });
         this.entity.camera.renderTarget = renderTarget;
         this.entity.camera.camera.horizontalFov = width < height;
+
+        // create pick mode render target
+        this.pickModeColorBuffer = createTexture(width, height, pixelFormat);
+        this.pickModeRenderTarget = new RenderTarget({
+            colorBuffer: this.pickModeColorBuffer,
+            depth: false,
+            flipY: false,
+            samples: samples,
```

Comment:
I assume the picker rendering here renders some IDs. Is it compatible with multisampling? I assume the resolve would average IDs.

Reply:
probably not compatible with resolve, but as mentioned above it's disabled. i might completely remove multisampling support in the next update.